src/Symfony/Bridge/Twig/AppVariable.php

```diff
@@ -180,4 +180,26 @@ public function getFlashes($types = null)
 
         return $result;
     }
+
+    /**
+     * Return a environment variable (defined in .env for example).
+     *
+     * If the $name is null or empty, null is returned.
+     *
+     * @return string|null
+     */
+    public function getEnv($name = null)
+    {
+        if (null === $name || '' === $name) {
+            return null;
+        }
+        
+        if (null === $this->requestStack) {
+            throw new \RuntimeException(sprintf('The app.request variable is not available'));
+        }    
+        
+        $request = $this->getRequest();
```

Comment:
I don't think we should use the request for this: this would mean we can't use it in console commands where environment variables are totally valid.

Reply:
Yes, I need to look at how we can find the env variables without using `Request` in `AppVariable`.